cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json

```diff
@@ -1,7 +1,7 @@
12/5e33/v4.11/HLT/V2</a></td></tr>\n      <tr><td>190482 - 190539</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_3_patch1\">CMSSW_5_2_3_patch1</a></td><td><a href=\"/record/6134\">/cdaq/physics/Run2012/5e33/v4.4/HLT/V7</a></td></tr>\n      <tr><td>190661 - 190679</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_3_patch1\">CMSSW_5_2_3_patch1</a></td><td><a href=\"/record/6135\">/cdaq/physics/Run2012/5e33/v4.7/HLT/V1</a></td></tr>\n      <tr><td>194270 - 194317</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch3\">CMSSW_5_2_4_onlpatch3</a></td><td><a href=\"/record/6136\">/cdaq/physics/Run2012/7e33/v2.2/HLT/V2</a></td></tr>\n      <tr><td>195868 - 195950</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch3\">CMSSW_5_2_4_onlpatch3</a></td><td><a href=\"/record/6137\">/cdaq/physics/Run2012/7e33/v2.4/HLT/V1</a></td></tr>\n      <tr><td>193334 - 193336</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch1\">CMSSW_5_2_4_onlpatch1</a></td><td><a href=\"/record/6138\">/cdaq/physics/Run2012/5e33/v4.16/HLT/V18</a></td></tr>\n      <tr><td>193998 - 194210</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch3\">CMSSW_5_2_4_onlpatch3</a></td><td><a href=\"/record/6139\">/cdaq/physics/Run2012/7e33/v2.1/HLT/V12</a></td></tr>\n      <tr><td>190591 - 190595</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_3_patch1\">CMSSW_5_2_3_patch1</a></td><td><a href=\"/record/6140\">/cdaq/physics/Run2012/5e33/v4.5/HLT/V3</a></td></tr>\n      <tr><td>206859 - 207100</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_7_onlpatch2\">CMSSW_5_2_7_onlpatch2</a></td><td><a href=\"/record/6141\">/cdaq/physics/Run2012/8e33/v2.0/HLT/V3</a></td></tr>\n      <tr><td>199960 - 200381</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_6_onlpatch2\">CMSSW_5_2_6_onlpatch2</a></td><td><a href=\"/record/6142\">/cdaq/physics/Run2012/7e33/v4.0/HLT/V4</a></td></tr>\n      <tr><td>207214 - 207518</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_7_onlpatch3\">CMSSW_5_2_7_onlpatch3</a></td><td><a href=\"/record/6143\">/cdaq/physics/Run2012/8e33/v2.1/HLT/V5</a></td></tr>\n      <tr><td>193093 - 193124</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch1\">CMSSW_5_2_4_onlpatch1</a></td><td><a href=\"/record/6144\">/cdaq/physics/Run2012/5e33/v4.16/HLT/V14</a></td></tr>\n      <tr><td>199318 - 199608</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch5\">CMSSW_5_2_4_onlpatch5</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2012/7e33/v3.1/HLT/V2</a></td></tr>\n      <tr><td>200466 - 202504</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_6_onlpatch3\">CMSSW_5_2_6_onlpatch3</a></td><td><a href=\"/record/6146\">/cdaq/physics/Run2012/7e33/v4.1/HLT/V1</a></td></tr>\n      </tbody>\n      </table>\n      "
```

Comment:
BTW for Run2011B datasets, in the provenance part, we still use terms such as "Configuration file for RECO step reco_2011A_BTag", pointing to records marked as "Configuration file for RECO step reco_2011A_BTag ". Some people may be confused that we seem advertise only 2011A everywhere.  

Example: http://opendata-dev.cern.ch/record/270 pointing to http://opendata-dev.cern.ch/record/3603

Is this OK? We could perhaps leave out "A" if it makes sense...

Reply:
Yes, dropping A would make sense as the name is ours in any case. Thanks for spotting that.

Reply:
OK, dropped A everywhere, from Run2011A and Run2011B records.